File: conference/registration-bc/web/src/main/java/com/microsoft/conference/registration/web/Models/OrderSeatsViewModel.cs
using System;
using System.Collections.Generic;
using Registration.ReadModel;

namespace Registration.Web.Models
{
    public class OrderSeatsViewModel
    {
        public OrderSeatsViewModel()
        {
            this.SeatAssignments = new List<OrderSeatAssignment>();
        }

        public Guid OrderId { get; set; }
        public IList<OrderSeatAssignment> SeatAssignments { get; set; }
    }
}

File: samples/conference/registration-bc/src/main/java/com/microsoft/conference/registration/controller/OrderSeatsViewModel.java
package com.microsoft.conference.registration.controller;

import com.microsoft.conference.registration.readmodel.service.OrderSeatAssignmentVO;
import lombok.Data;

import java.util.List;

@Data
public class OrderSeatsViewModel {
    private String orderId;

    private List<OrderSeatAssignmentVO> seatAssignments;
}
